tests/service/views/test_project_views.py

```diff
@@ -119,7 +119,9 @@ def test_get_lock_status_unlocked(svc_client_setup):
     """Test getting lock status for an unlocked project."""
     svc_client, headers, project_id, _, _ = svc_client_setup
 
-    response = svc_client.get("/1.0/project.lock_status", query_string={"project_id": project_id}, headers=headers)
+    response = svc_client.get(
+        "/1.0/project.lock_status", query_string={"project_id": project_id}, headers=headers, content_type="text/xml"
```

Comment:
Is `content_type="text/xml"` to test that it works with any content_type? Shouldn't it work without setting a content type?

Reply:
It is for having a failing test. The default content type set by the flask test client is `application/json` and this test won't fail if the endpoint has `@accepts_json` decorator.

Reply:
ah I see